components/admin/components/AddResourceModal.tsx

```diff
@@ -51,17 +51,7 @@ const AddResourceModal = ({
 		useLocation(city, country)
 
 	useEffect(() => {
-		if (country === 'GB') {
-			setState('England')
-		} else if (country === 'AU') {
-			setState('Northern Territory')
-		} else if (country === 'US') {
-			setState('Alabama')
-		} else if (country === 'NZ') {
-			setState('Marlborough')
-		} else {
-			setState('Alberta')
-		}
+		setState('No State / Province')
```

Comment:
I don't think this needs to be changed at all.

Reply:
Okay I wasn't sure if we wanted it to default to the No state option. I can change that